### PR TITLE
Add styles for app

### PR DESCRIPTION
**Pull request by marvin-nyalik**
In this branch:

- I added detailed styling for the application using the Zeplin design template.
- Matched my final styles with the design

Let me know of required changes.

### CODE REVIEW

File: src/components/info.module.css [hunk 1]
```diff
@@ -146,3 +146,49 @@
...
+    width: 100%;
+    border-bottom: 1px solid lightgray;
+  }
+
+  .progress {
+    width: 20%;
+    border: none;
+    height: 15vh;
+    display: flex;
+  }
```
Comment (OyePriscilla) [inline src/components/info.module.css:198]: Kindly place the percentage text out of the progress label and don't forget to add completed text. ℹ️

File: src/components/addBook.module.css [hunk 2]
```diff
@@ -57,3 +57,32 @@
...
+  }
+
+  .heading {
+    margin: 5px auto;
+  }
+
+  .form {
+    flex-direction: column;
+    margin: 0 auto;
+  }
```
Comment (OyePriscilla) [inline src/components/addBook.module.css:86]: - [ ] Kindly implement correct background-color in accordance with the design (the body element inclusive). 👍

File: src/components/addBook.module.css [hunk 3]
```diff
@@ -57,3 +57,32 @@
...
+
+@media screen and (max-width: 768px) {
+  .cover {
+    width: 90vw;
+    margin: 10px auto;
+  }
+
+  .heading {
+    margin: 5px auto;
+  }
```
Comment (OyePriscilla) [inline src/components/addBook.module.css:81]: - [ ] I observed that there is difference between your design and the given template. It's stated in the requirement that design should be close to Zeplin's design. Kindly adjust the margin around the `nav` items, design the `remove` button expected and the `hr` element that is missing.

## Template
Untitled

## Your Design

Untitled

File: src/components/info.module.css [hunk 4]
```diff
@@ -146,3 +146,49 @@
...
+
+  .chapter {
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+    justify-content: flex-start;
+    margin: 0;
+    border-top: 1px solid lightgray;
+    padding-top: 20px;
+  }
```
Comment (OyePriscilla) [inline src/components/info.module.css:208]: - [ ] Kindly increase the left margin in accordance to the template. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @marvin-nyalik 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
All elements in design added 👍 
Styling is **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/) ❎ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @marvin-nyalik 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- All elements in design added 👍
- Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) ❎
- Well-structured files :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**